src/databricks/labs/ucx/account/workspaces.py

```diff
@@ -51,8 +51,12 @@ def workspace_clients(self, workspaces: list[Workspace] | None = None) -> list[W
             workspaces = self.get_accessible_workspaces()
         clients = []
         for workspace in workspaces:
-            ws = self.client_for(workspace)
-            clients.append(ws)
+            try:
+                ws = self.client_for(workspace)
+            except PermissionDenied as e:
+                logger.warning(f"Cannot get a workspace client for: {workspace.deployment_name}", exc_info=e)
+            else:
+                clients.append(ws)
```

Comment:
```suggestion
                ws = self.client_for(workspace)
                clients.append(ws)
            except PermissionDenied as e:
                logger.warning(f"Cannot get a workspace client for: {workspace.deployment_name}", exc_info=e)
```
didn't know `else:` branch exists for `try:`